Source/dom/style/Text.js

```diff
@@ -184,7 +188,12 @@ export function defineTextStyleProperties(Style) {
       }
 
       updateParagraphStyle(this._object, (paragraphStyle) => {
-        const translated = TextAlignmentMap[mode]
+        // Get a platform independent raw text alignment value
+        // NSTextAlignment is represented differently on Intel and M1 chips
+        const translated =
+          TextAlignmentMap[
+            MSTextAlignmentConverter.archiveNSTextAlignment(mode)
```

Comment:
@klaaspieter it doesn't seem to be working on M1: `mode` appears to be a string here (e.g. `"center"`), maybe we need to pass `TextAlignmentMap[mode]` to `archiveNSTextAlignment` instead? (The other way around)

Reply:
Should be fixed now 🤞🏻